src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.test.tsx

```diff
@@ -52,7 +52,7 @@ describe("MeasureInformation component", () => {
 
   it("should render the component with measure's name populated", () => {
     const { getByTestId } = render(<MeasureInformation />);
-    const result: HTMLElement = getByTestId("measureNameEdit");
+    const result: HTMLElement = getByTestId("measure-name-edit");
```

Comment:
We need some common pattern for naming IDs.

Reply:
I will add this to the guidelines document.